example/src/App.tsx

```diff
@@ -116,6 +116,7 @@ function App() {
                 inputType={inputType}
                 renderInput={(props) => <input {...props} />}
                 shouldAutoFocus
+                shouldRemoveWhiteSpaceOnPaste
```

Comment:
Can you also add documentation for this in the Readme?

Reply:
Done